spec/motion-spec/should_spec.rb

```diff
@@ -1,5 +1,247 @@
+  it 'has should.include' do
+    proc { [1, 2, 3].should.include 2 }.should succeed
+    proc { [1, 2, 3].should.include 4 }.should fail
+
+    proc { { 1 => 2, 3 => 4 }.should.include 1 }.should succeed
+    proc { { 1 => 2, 3 => 4 }.should.include 2 }.should fail
+  end
+
+  it 'has should.be.a.kind_of' do
+    proc { Array.should.be.kind_of Module }.should succeed
+    proc { 'string'.should.be.kind_of Object }.should succeed
+    proc { 1.should.be.kind_of Comparable }.should succeed
+
+    proc { Array.should.be.a.kind_of Module }.should succeed
+
+    proc { 'string'.should.be.a.kind_of Class }.should fail
+  end
+
+  it 'has should.match' do
+    proc { 'string'.should.match(/strin./) }.should succeed
+    proc { 'string'.should =~ /strin./ }.should succeed
+
+    proc { 'string'.should.match(/slin./) }.should fail
+    proc { 'string'.should =~ /slin./ }.should fail
+  end
+
+  it 'has should.not.raise' do
+    proc { proc { 1 + 1 }.should.not.raise }.should succeed
+    proc { proc { 1 + 1 }.should.not.raise(Interrupt) }.should succeed
+
+    proc {
+      proc {
+        proc {
+          Kernel.raise ZeroDivisionError.new('ArgumentError')
+        }.should.not.raise(RuntimeError, Comparable)
+      }.should.raise ZeroDivisionError
+    }.should succeed
+
+    proc { proc { raise 'Error' }.should.not.raise }.should fail
+  end
+
+  it 'has should.throw' do
+    proc { proc { throw :foo }.should.throw(:foo) }.should succeed
+    proc { proc {       :foo }.should.throw(:foo) }.should fail
+
+    should.throw(:foo) { throw :foo }
+  end
+
+  it 'has should.not.satisfy' do
+    proc { should.not.satisfy { 1 == 2 } }.should succeed
+    proc { should.not.satisfy { 1 == 1 } }.should fail
+  end
+
+  it 'has should.not.equal' do
+    proc { 'string1'.should.not.eq 'string2' }.should succeed
+    proc { 'string1'.should.not.eq 'string1' }.should fail
+  end
+
+  it 'has should.not.match' do
+    proc { 'string'.should.not.match(/sling/) }.should succeed
+    proc { 'string'.should.not.match(/string/) }.should fail
+    # proc { "string".should.not.match("strin") }.should fail
+
+    proc { 'string'.should.not =~ /sling/ }.should succeed
+    proc { 'string'.should.not =~ /string/ }.should fail
+    # proc { "string".should.not =~ "strin" }.should fail
+  end
+
+  it 'has should.be.identical_to/same_as' do
+    proc { s = 'string'; s.should.be.identical_to s }.should succeed
+    proc { 'string'.should.be.identical_to 'string' }.should fail
+
+    proc { s = 'string'; s.should.be.same_as s }.should succeed
+    proc { 'string'.should.be.same_as 'string' }.should fail
+  end
+
+  it 'has should.respond_to' do
+    proc { 'foo'.should.respond_to :to_s }.should succeed
+    proc { 5.should.respond_to :to_str }.should fail
+    proc { :foo.should.respond_to :nx }.should fail
+  end
+
+  it 'has should.be.close' do
+    proc { 1.4.should.be.close 1.4, 0 }.should succeed
+    # TODO this one is disabled because it will probably never run on MacRuby.
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.